src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1556,7 +1675,7 @@ private void ReportUnusedVariables(ImmutableArray<LocalSymbol> locals)
             }
         }
 
-        private void ReportIfUnused(LocalSymbol symbol, bool assigned)
+        protected virtual void ReportIfUnused(LocalSymbol symbol, bool assigned)
```

Comment:
>protected [](start = 8, length = 9)

Can you please explain this change? If it is a vestige of previous refactorings, please revert.